src/router/cloudRouter.ts

```diff
@@ -81,13 +88,23 @@ const dashboardLayout = createRoute({
 	getParentRoute: () => rootRoute,
 	id: '_dashboardLayout',
 	component: Dashboard,
+	beforeLoad: ({ context, location }) => {
+		if (!context.authentication.isLoading && !context.authentication.user) {
+			throw redirect({
+				to: '/',
+				search: {
+					redirect: location.href,
```

Comment:
We should make sure this works with more complex routes, I got a bit blocked by central manager changes for creating clusters 😮‍💨